### DISCUSSION TITLE
Add a slash command to insert all the visible tabs into the assistant panel (`/tab visible`)

### DISCUSSION THREAD

**Discussion by dlants** [zed-industries/zed#37493] [Feature Requests]
Summary: Provide a way for `/tab` to include all visible tabs (e.g. `/tab visible`)

When I'm working on several files in parallel, I often open them in a vsplit. Using `/tab` currently in that context is a bit confusing, since it's hard to anticipate which of the vsplit files will be inserted into the assistant context. I think it would be really helpful if /tab inserted all of the visible panes into the context.

`/tab all` is not really what I expect either, since there may be many tabs in the background, and that's more than I typically intend.

### Environment

Zed: v0.161.1 (Zed)
OS: macOS 14.6.1
Memory: 16 GiB
Architecture: aarch64

**Comment by notpeter** [comment]
Would something like `/tab visible` (adds the currently selected/visible tab from each pane) satisfy your request?

Similarly maybe `/tab pane` could be a halfway point between `/tab` and `/tab all` and only include all tabs from the most recently focused pane?

**Comment by dlants** [comment]
These both sound like good ideas. I think there's a need in here too for making it more clear "which pane will be imported?" for the /tab and /tab pane command, if you have multiple panes visible.
